src/envs/pybullet_blocks.py

```diff
@@ -29,6 +29,10 @@ class PyBulletBlocksEnv(PyBulletEnv, BlocksEnv):
     _table_pose: ClassVar[Pose3D] = (1.35, 0.75, 0.0)
     _table_orientation: ClassVar[Sequence[float]] = [0., 0., 0., 1.]
 
+    # Robot parameters.
+    _ee_orn = p.getQuaternionFromEuler([0.0, np.pi / 2, -np.pi])
```

Comment:
do these need to be implemented by all subclasses of pybullet env? if so, can we make them abstract properties in the pybullet env?

Reply:
we don't need to, it's correct as is. because `create_pybullet_robot` is abstract already, and that's where the subclasses use this `ee_orn` thing. it's not used in the pybullet base env.